e2e/test/scenarios/auditing/ad-hoc.cy.spec.js

```diff
@@ -36,4 +36,41 @@ describeEE("audit > ad-hoc", () => {
       cy.get(".ace_content").contains("SELECT 123");
     });
   });
+
+  describe("notebook query", () => {
+    beforeEach(() => {
+      cy.intercept('/api/dataset').as('dataset');
+
+      restore();
+
+      cy.log("Run ad hoc notebook query as normal user");
+      cy.signInAsNormalUser();
+      startNewQuestion();
+      cy.findByTextEnsureVisible("Sample Database").click();
+      cy.findByTextEnsureVisible("Orders").click();
+
+      cy.button('Visualize').click();
+
+      cy.wait('@dataset');
```

Comment:
@iethree `openOrdersTable()` is doing exactly this thing without all the intermediate steps

Reply:
cool! I'm working on the fix now, so I'll update the repro when I unskip it!

Reply:
https://github.com/metabase/metabase/pull/29525/files#diff-f0f37f54acccc33b9241549ae4ef8387feb4e359e0cefe2b02171a2456c08159L48-R48